charts/v2.16.0/cray-hms-hmcollector/.gitignore

```diff
@@ -0,0 +1,5 @@
+# by default we'll ignore any subcharts included, but simply adjust this if need be
```

Comment:
Is charts/v2.16.0 the right directory name? I expected charts/v2.16 instead?

Reply:
You are right. Moved it to the right spot.